mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -212,11 +261,22 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
         self.best_feature_names_ = None
         self.best_score_ = None
 
+        self.fixed_features = self.fixed_features
+        self.fixed_features_set_ = set()
```

Comment:
You can move this line 265 to line 277 just before when you want to use it. So, you can help reader follow you code more easily.